packages/teams-js/src/public/teamsAPIs.ts

```diff
@@ -58,7 +58,7 @@ export namespace teamsCore {
    */
   export function registerOnLoadHandler(handler: registerOnLoadHandlerFunctionType): void {
     registerOnLoadHandlerHelper(handler, () => {
-      if (!isNullOrUndefined(handler) && !isSupported()) {
+      if (!isNullOrUndefined(handler)) {
```

Comment:
Hi @kpolnitz , one question about this change based on what you wrote in the description: 
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/18446904/d84b0262-02d3-4f7d-a53c-fdddee04f681)
How does this change make the function throw less often? It seems like removing the isSupported check would throw more often because now it looks like it will throw any time the handler is defined.